images/base/Dockerfile

```diff
@@ -88,7 +88,7 @@ RUN containerd --version \
 
 # Install CNI binaries to /opt/cni/bin
 # TODO(bentheelder): doc why / what here
-ARG CNI_VERSION="0.7.5"
+ARG CNI_VERSION="0.8.2"
```

Comment:
this doesn't work
see https://github.com/kubernetes-sigs/kind/pull/875

Reply:
yeah :sweat_smile: